develop/toolkit/api-libraries/sidecar.md

```diff
@@ -1,4 +1,100 @@
 ---
 title: Sidecar Rest API
-description: TODO
----
\ No newline at end of file
+description: Learn about Substrate API Sidecar, a REST service that provides endpoints for interacting with Polkadot SDK-based chains and simplifies blockchain interactions.
+---
+
+# Sidecar API
+
+## Introduction
+
+The [Sidecar Rest API](https://github.com/paritytech/substrate-api-sidecar){target=\_blank} is a service that provides a REST interface for interacting with Polkadot SDK-based blockchains. With this API, developers can easily access a broad range of endpoints for nodes, accounts, transactions, parachains, and more.
+
+Sidecar functions as a caching layer between your application and a Polkadot SDK-based node, offering standardized REST endpoints that simplify interactions without requiring complex, direct RPC calls. This approach is especially valuable for developers who prefer REST APIs or build applications in languages with limited WebSocket support.
+
+Some of the key features of the Sidecar API include:
+
+- **REST API Interface** - provides a familiar REST API interface for interacting with Polkadot SDK-based chains
```

Comment:
based on a quick look, for the list items here. on the left, only the first word needs to be capitalized

Reply:
fixed on [9afc1a8](https://github.com/polkadot-developers/polkadot-docs/pull/141/commits/9afc1a867dc554a740304603430fc6c792acc5c1)! Thank you!!